main.go

```diff
@@ -43,6 +44,8 @@ func init() {
 
 	utilruntime.Must(lokiv1beta1.AddToScheme(scheme))
 	// +kubebuilder:scaffold:scheme
+
+	utilruntime.Must(monitoringv1.AddToScheme(scheme))
```

Comment:
Let's put a `TODO` here to make this loading optional via a `loki-operator` CLI flag.